### PR TITLE
Change the clock UI using a new file "style2.css"

### PR DESCRIPTION
**Pull request by JacobJax**
Checkout the changes i've made on the Clock's UI

### CODE REVIEW

File: digital-clock/style2.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    align-content: center;
+    justify-content: center;
+    padding: 50px;
+    background-color: #fff;
+}
+.banner-content{
+    padding: 15px;
+    background-color: #fff;
+    box-shadow: -3px -3px 7px #f3f3f3dc, 3px 3px 5px rgba(94, 104, 121, .288);
+    border-radius: 5px;
```
Comment (gbrowns) [inline digital-clock/style2.css:18]: Hows this block suppose to fit-up

**Review by gbrowns — APPROVED** [review]
Nice improvements on the UI, Jax

File: digital-clock/style2.css (hunk 1 above)
Comment (JacobJax) [reply digital-clock/style2.css:18]: I added the .profile div and the .time div to the .banner.content div
```suggestion
.banner-content{
    padding: 15px;
    background-color: #fff;
    box-shadow: -3px -3px 7px #f3f3f3dc, 3px 3px 5px rgba(94, 104, 121, .288);
    border-radius: 5px;
```

**Comment by gbrowns** [comment]
THis is amazing Jax

**Comment by gbrowns** [comment]
Thanks Jax...i had skimmed it up and confirmed how everything works.its
actually amazing

**Pull request merged**